### ISSUE TITLE
Your service has missing or invalid owners

### ISSUE THREAD

**Issue by shopify-services** [Shopify/rbi#3]
# Your service has missing or invalid owners

## TL;DR
Make sure you have at least one owner listed in your service.yml and that every owner points to a valid GitHub team slug or user login.

## Why is this being asked?
As we work toward revamping the way ownership works here, we need to ensure we have a good picture of the current state of affairs. For most services this is obvious (ServicesDB is owned by Shopify/production-excellence, which maps to a Production Excellence team in Vault). For some services however, it is very much not obvious, as they have no entries or invalid entries in their service.yml, making it difficult to determine the best path forward.

## What will happen if it doesn't get done within the expected timeframe?
As we attempt to infer the correct owner, we may end up misclassifying things by putting ownership of a service onto the wrong team. It would be easily remedied, but still wouldn't be the optimal experience.

## This doesn't apply to my service. What do I do?
First, leave a comment explaining why it doesn't apply. Then, leave another comment as `/not_applicable`, and close the issue.
If you change your mind, make sure to comment any reason and reopen the issue. Once you've fixed it, services-db will close the issue for you next day.

## Detailed description
For more information about why we're making this change, please see this project brief here: https://docs.google.com/document/d/1KKosigiJhy2DGmc22jJ8l3-Tumkl53dzvgt1Tuw1TRw/edit.

## I have questions/concerns about this
Please contact the Production Excellence team using #core-build-manage.

**Your service**: [rbi/production](https://services.shopify.io/services/rbi/production)
